drake/systems/plants/RigidBodySystem.h

```diff
@@ -224,12 +229,20 @@ class DRAKERBSYSTEM_EXPORT RigidBodySystem {
   friend DRAKERBSYSTEM_EXPORT StateVector<double> getInitialState(
       const RigidBodySystem& sys);
 
+  /*!
+   * Returns a string describing the semantics of the state vector.
+   */
+  std::string getStateVectorSemantics() const;
```

Comment:
Oops this was a vestige of my debugging printouts. My intent was to provide a description of each element in the state vector. Example information includes state name and unit of measurement.

In general, I believe the more semantic metadata included with each variable / parameter the better since it can increase interoperability and reduce the possibility of bugs. At a minimum, it can prevent conflating variables of incompatible types (e.g., comparing a position to a velocity). Related paper: https://pdfs.semanticscholar.org/fc3a/68b46faf209532f4434f41208e39493dc3b5.pdf